app/src/androidTest/java/ch/epfl/sweng/erpa/activities/live/LoginActivityTest.java

```diff
@@ -0,0 +1,4 @@
+package ch.epfl.sweng.erpa.activities.live;
+
+public class LoginActivityTest {
+}
```

Comment:
Finish this please

Reply:
This file should never had been commited. It was for debugging